presto-tests/src/test/java/com/facebook/presto/server/TestQueryResource.java

```diff
@@ -96,12 +96,14 @@ public void testGetQueryInfos()
 
         infos = getQueryInfos("/v1/query?limit=5");
         assertEquals(infos.size(), 5);
-        assertEquals(infos.get(0).getQuery(), "SELECT * from tpch.sf100.orders -- 1");
-        assertEquals(infos.get(1).getQuery(), "SELECT * from tpch.sf100.orders -- 2");
-        assertEquals(infos.get(2).getQuery(), "SELECT * from tpch.sf100.orders -- 3");
-        assertEquals(infos.get(3).getQuery(), "SELECT 3");
-        assertEquals(infos.get(4).getQuery(), "SELECT x FROM y");
-        assertStateCounts(infos, 0, 1, 3, 1);
+        // It seems that reliance on the sleep and query timing don't work together too well.
```

Comment:
I think it is best to disable the entire test .
Running locally I can see failures as well on line 92 above.

Reply:
Local runs fail for me all the time, even before the last change.
I think it is a very flaky test.